papers.html

```diff
@@ -301,7 +321,7 @@ <h5>de Wit et al. 2024</h5>
         <div class="card card-body">
           <p><b>Authors: </b> de Wit, S.; Bonanos, A. Z.; Antoniadis, K.; Zapartas, E.; Ruiz, A.; Britavskiy, N.; Christodoulou, E.; De, K.; Maravelias, G.; Munoz-Sanchez, G.; Tsopela, A.</p>
           <p><b>Access: </b> <a href="https://arxiv.org/abs/2402.12442" target="_blank">arXiv:2403.2402.12442</a> || <a href="https://ui.adsabs.harvard.edu/abs/2024arXiv240212442D/abstract" target="_blank">NASA ADS</a> 
```

Comment:
`arXiv:2403.2402.12442` should read `arXiv:2402.12442`

Reply:
```suggestion
          <p><b>Access: </b> <a href="https://arxiv.org/abs/2402.12442" target="_blank">arXiv:2402.12442</a> || <a href="https://ui.adsabs.harvard.edu/abs/2024arXiv240212442D/abstract" target="_blank">NASA ADS</a> 
```